docs/source/tracking.rst

```diff
@@ -722,6 +720,7 @@ then make API requests to your remote tracking server.
         mlflow_set_experiment("/my-experiment")
         mlflow_log_param("a", "1")
 
+Note that to use the artifact storage capabilities of a remote server, the client need a direct access to the artifact store. The client is the one pushing directly artifacts to the artifact storage, and not through the tracking server. How to setup credentials for the artifact storage depends on the :ref:`Artifact Stores <artifact-stores>` technology setup for the server. Note that some technology choices impose that credential will transit in clear text on the network.
```

Comment:
Any reason to put this here, rather than in the Artifact Store section?

Reply:
(also I think the test failure may be because of an invalid ref)

Reply:
the reason was your own suggestion in the original issue ([here](https://github.com/mlflow/mlflow/issues/2834#issuecomment-639892606)) :wink: 

yes it may be that my removal of the duplicated sections break things... is there a way to go for solving this ?

Reply:
Ha, my mistake. This is a reasonable place to put this note. 

For debugging the failure, I think you can run Sphinx locally with this command: https://github.com/mlflow/mlflow/blob/master/.circleci/config.yml#L33